frontend/mr-admin-flate/src/components/tiltaksgjennomforinger/TiltaksgjennomforingslisteForTiltakstyper.tsx

```diff
@@ -0,0 +1,53 @@
+import { Tiltaksgjennomforingrad } from "./Tiltaksgjennomforing";
+import { useTiltaksgjennomforinger } from "../../api/tiltaksgjennomforing/useTiltaksgjennomforinger";
+import { Alert, Heading, Loader } from "@navikt/ds-react";
+import { Tiltaksgjennomforing } from "mulighetsrommet-api-client";
+import styles from "./TiltaksgjennomforingslisteForTiltakstyper.module.scss";
+import tiltaksgjennomforingsStyles from "./Tiltaksgjennomforingeroversikt.module.scss";
+
+interface TiltaksgjennomforingerListeTiltakstypeProps {
```

Comment:
Så lenge det bare er et interface for Props i komponenten kan du gjerne bare kalle interface for `Props` så er vi konsekvente i kodestilen